addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5326,20 +5302,93 @@ registry.layout_column = SnippetOptionWidget.extend({
         }
         // Removing the grid properties.
         delete rowEl.dataset.rowCount;
-        rowEl.style.removeProperty('--grid-item-padding-x');
+        rowEl.style.removeProperty('--grid-item-padding-x'); // Kept for compatibility.
         rowEl.style.removeProperty('--grid-item-padding-y');
     },
+    /**
+     * Removes all the eventual grid previews, to avoid unwanted behaviors (when
+     * toggling normal mode for example).
+     *
+     * @private
+     */
+    _clearAllPreviews(columnEls) {
+        // Removing the padding previews.
+        for (const columnEl of columnEls) {
+            if (columnEl._removePaddingPreview) {
+                columnEl._removePaddingPreview();
```

Comment:
I don't want something like that to be attached to the column elements. I guess the right way depends on the answer to my previous comment ("can't you rely on cleanUI?")... :grimacing: